frontend/packages/topology/src/components/graph-view/components/nodes/BaseNode.tsx

```diff
@@ -115,6 +120,7 @@ const BaseNode: React.FC<BaseNodeProps> = ({
         <DefaultNode
           className={classNames('odc-base-node', className, {
             'is-filtered': filtered,
+            [StatusModifier[alertVariant]]: showAlertStatus,
```

Comment:
The `DefaultNode` will set this automatically if we calculate the right `nodeStatus`

```suggestion
```